packages/api/src/services/synthetic-prices.ts

```diff
@@ -49,24 +58,99 @@ export default function (config: Config, appState: Dependencies) {
     return Promise.allSettled(addresses.map(updatePriceHistory));
   }
 
-  async function updateLatestPrice(empAddress: string) {
+  // updates raw synth price, in relation to collateral
+  async function updateLatestSynthPrice(empAddress: string) {
```

Comment:
this update adds another type of synthetic price, originally it was just the price pulled from the bot price feed, but in order to make this useful, we also have to convert it to USD. Thats what this PR adds, is the raw price conversion to usd.